src/popupmenu.c

```diff
@@ -941,18 +1023,7 @@ pum_redraw(void)
 	    screen_fill(row, row + 1, col, pum_col + pum_width, ' ', ' ',
 								orig_attr);
 	if (pum_scrollbar > 0)
-	{
-#ifdef FEAT_RIGHTLEFT
-	    if (pum_rl)
-		screen_putchar(' ', row, pum_col - pum_width,
-			i >= thumb_pos && i < thumb_pos + thumb_height
-						  ? attr_thumb : attr_scroll);
-	    else
-#endif
-		screen_putchar(' ', row, pum_col + pum_width,
-			i >= thumb_pos && i < thumb_pos + thumb_height
-						  ? attr_thumb : attr_scroll);
-	}
+	    pum_draw_scrollbar(row, i, thumb_pos, thumb_height);
```

Comment:
pum_draw_scrollbar() already checks for `pum_scrollbar`. So you can also get rid of the condition on line 1025